trl/models/modeling_sd_base.py

```diff
@@ -24,7 +24,14 @@
 from diffusers.loaders import AttnProcsLayers
 from diffusers.models.attention_processor import LoRAAttnProcessor
 from diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion import rescale_noise_cfg
-from diffusers.utils.torch_utils import randn_tensor
+
+from ..import_utils import is_diffusers_greater_than
+
+
+if is_diffusers_greater_than("0.20.2"):
+    from diffusers.utils.torch_utils import randn_tensor
+else:
+    from diffusers.utils import randn_tensor
```

Comment:
If I understood Patrick correctly, `randn_tensor` is an internal method that could be deprecated, delete, renamed at any time. Should we not use it and instead add our own?

Reply:
Makes sense yeah !